templates/config.yml

```diff
@@ -85,20 +85,25 @@ gateways:
 # but you are free to create your own and put a fully qualified class name below. The order is,
 # of course, important.
 blockchain_adapters:
-  - BlockchainInfo # uses Straight::Blockchain::BlockchainInfoAdapter
-  - Mycelium       # uses Straight::Blockchain::MyceliumAdapter
-  # - InsightAdapter # uses Straight::Blockchain::InsightAdapter
-  # - MyCustomClass::MyCustomBlockchainAdapter
-
-# If Insight adapter is enabled, it is necessary to provide a url for it.
-#   insight_url: "https://localhost/api"
-# You can also set a testnet insight url to use:
-#   insight_test_url: "https://test.localhost/api"
-
-# Websocket Insight Url
-# Uncomment it and put real Insight server address, for using fast transaction tracking. It works only
-# for gateways and orders without confirmation required (eq to 0).
-# insight_websocket_url: "wss://localhost:4000"
+  # Uses Straight::Blockchain::BlockchainInfoAdapter
+  - name: BlockchainInfo
+  # Uses Straight::Blockchain::MyceliumAdapter
+  - name: Mycelium
+  # Uses Straight::Blockchain::InsightAdapter
+  - name: Insight
+    # If Insight adapter is enabled, it is necessary to provide a url for it.
+    main_url: 'https://localhost/api'
```

Comment:
Yes, let's use option names that are familiar to Bitcoin users: `mainnet_url` and `testnet_url`. @AlexeyShepelev you should fix this.

Also why

    - name: Insight
      main_url: 'https://localhost/api'

Instead of

    Insight:
      main_url: 'https://localhost/api'

?